src/Storage/Entity/ContentRouteTrait.php

```diff
@@ -31,6 +32,10 @@ public function editlink()
     {
         $perm = 'contenttype:' . $this->contenttype['slug'] . ':edit:' . $this->id;
 
+        if (empty($this->app)) {
+            $this->app = AppSingleton::get();
```

Comment:
Must not be used, ever :wink:

Reply:
Hmmm I swear I copied this from somewhere else in Bolt :thinking:. Doesn't matter, I'll update all of these to remove the `empty`!

Reply:
Ah, it is not the `empty()` but the use of the application object in the entity … we'll chat about it in Slack in the coming week when I am back in NL

Reply:
Not the `empty()`?! :astonished: I never thought I'd hear you say that :stuck_out_tongue_winking_eye:. Okay sure, let's chat once your back from NL.